### PR TITLE
Add dirty code

### PR DESCRIPTION
**Pull request by pocke**

### CODE REVIEW

File: test.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+a = 1
```
Comment (sumyapp) [inline test.rb:1]: **[rubocop]**
- Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

Comment (wata727) [inline test.rb:1]: **[staging][rubocop]**
- Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/43/diff/24438581ae061ef232184aaf36010609bacd278a?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/43/diff/24438581ae061ef232184aaf36010609bacd278a?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

File: test.rb (hunk 1 above)
Comment (sumyapp) [inline test.rb:1]: **[rubocop]**
- Missing frozen string literal comment.

Comment (wata727) [inline test.rb:1]: **[staging][rubocop]**
- Missing frozen string literal comment.

File: test.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  class AAA
```
Comment (sumyapp) [inline test.rb:2]: **[rubocop]**
- Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
- Inconsistent indentation detected.

Comment (wata727) [inline test.rb:2]: **[staging][rubocop]**
- Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
- Inconsistent indentation detected.

File: test.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    def initialize
```
Comment (sumyapp) [inline test.rb:3]: **[rubocop]**
- Missing top-level class documentation comment.

Comment (wata727) [inline test.rb:3]: **[staging][rubocop]**
- Missing top-level class documentation comment.

File: test.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+      @a
+    end
+
```
Comment (sumyapp) [inline test.rb:6]: **[rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

Comment (wata727) [inline test.rb:6]: **[staging][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: test.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
+a = 1
+  class AAA
+    def initialize
+      @a
+    end
+  
+    def a
```
Comment (sumyapp) [inline test.rb:7]: **[rubocop]**
- Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

Comment (wata727) [inline test.rb:7]: **[staging][rubocop]**
- Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

File: test.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+      @a
```
Comment (sumyapp) [inline test.rb:8]: **[rubocop]**
- Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

Comment (wata727) [inline test.rb:8]: **[staging][rubocop]**
- Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

File: test.rb [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+    end
+
```
Comment (sumyapp) [inline test.rb:10]: **[rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

Comment (wata727) [inline test.rb:10]: **[staging][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: test.rb [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    def initialize
+      @a
+    end
+  
+    def a
+      @a
+    end
+  
+      def foo
+        bar
```
Comment (sumyapp) [inline test.rb:12]: **[rubocop]**
- Inconsistent indentation detected.

Comment (wata727) [inline test.rb:12]: **[staging][rubocop]**
- Inconsistent indentation detected.

File: test.rb [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+      end
+  end
+
```
Comment (sumyapp) [inline test.rb:15]: **[rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

Comment (wata727) [inline test.rb:15]: **[staging][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: test.rb [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+    def a
+      @a
+    end
+  
+      def foo
+        bar
+      end
+  end
+  
+  a = ['x','y','z']
```
Comment (sumyapp) [inline test.rb:16]: **[rubocop]**
- Inconsistent indentation detected.
- Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)
- Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

Comment (wata727) [inline test.rb:16]: **[staging][rubocop]**
- Inconsistent indentation detected.
- Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)
- Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

File: test.rb [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+  puts a
```
Comment (sumyapp) [inline test.rb:17]: **[rubocop]**
- Inconsistent indentation detected.
- Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)

Comment (wata727) [inline test.rb:17]: **[staging][rubocop]**
- Inconsistent indentation detected.
- Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)

File: test.rb [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+
```
Comment (sumyapp) [inline test.rb:18]: **[rubocop]**
- 2 trailing blank lines detected. [:ref](https://github.com/bbatsov/ruby-style-guide#newline-eof)

Comment (wata727) [inline test.rb:18]: **[staging][rubocop]**
- 2 trailing blank lines detected. [:ref](https://github.com/bbatsov/ruby-style-guide#newline-eof)

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[Auto Correct by SideCI](https://sideci.com/gh/38860676/issues/pull_request/43/diff/308b45de796dd88abd3c1cb03f52324d002680b3?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://sideci.com/ja/docs/autofix-rubocop)

**Comment by wata727** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[staging] 
[Auto Correct by SideCI](https://www.codemagica.com/gh/38860676/issues/pull_request/43/diff/308b45de796dd88abd3c1cb03f52324d002680b3?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](https://www.codemagica.com/ja/docs/autofix-rubocop)

File: test.rb (hunk 1 above)
Comment (pocke) [inline test.rb:1]: **[development][rubocop]**
- Missing frozen string literal comment.

File: test.rb (hunk 2 above)
Comment (pocke) [inline test.rb:2]: **[development][rubocop]**
- Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
- Inconsistent indentation detected.

File: test.rb (hunk 3 above)
Comment (pocke) [inline test.rb:3]: **[development][rubocop]**
- Missing top-level class documentation comment.

File: test.rb (hunk 4 above)
Comment (pocke) [inline test.rb:6]: **[development][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: test.rb (hunk 5 above)
Comment (pocke) [inline test.rb:7]: **[development][rubocop]**
- Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

File: test.rb (hunk 6 above)
Comment (pocke) [inline test.rb:8]: **[development][rubocop]**
- Use `attr_reader` to define trivial reader methods. [:ref](https://github.com/bbatsov/ruby-style-guide#attr_family)

File: test.rb (hunk 7 above)
Comment (pocke) [inline test.rb:10]: **[development][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: test.rb (hunk 8 above)
Comment (pocke) [inline test.rb:12]: **[development][rubocop]**
- Inconsistent indentation detected.

File: test.rb (hunk 9 above)
Comment (pocke) [inline test.rb:15]: **[development][rubocop]**
- Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

File: test.rb (hunk 10 above)
Comment (pocke) [inline test.rb:16]: **[development][rubocop]**
- Inconsistent indentation detected.
- Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)
- Space missing after comma. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)

File: test.rb (hunk 11 above)
Comment (pocke) [inline test.rb:17]: **[development][rubocop]**
- Inconsistent indentation detected.
- Use `%w` or `%W` for an array of words. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-w)

File: test.rb (hunk 12 above)
Comment (pocke) [inline test.rb:18]: **[development][rubocop]**
- 2 trailing blank lines detected. [:ref](https://github.com/bbatsov/ruby-style-guide#newline-eof)

**Comment by pocke** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 
[development] 
[Auto Correct by SideCI](http://localhost:3000/gh/38860676/issues/pull_request/43/diff/308b45de796dd88abd3c1cb03f52324d002680b3?open_modal=1#rubocop)

See the documentation if you want to get more information about auto-correct.
[Document](http://localhost:3000/ja/docs/autofix-rubocop)

**Comment by pocke** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by pocke** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by pocke** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Comment by pocke** [comment]
Auto-correctable file didn't exist on this Pull Request from [SideCI](https://www.sideci.com/).

**Pull request closed without merging**